iothub/device/src/Serialization/SystemTestJsonPayloadConvention.cs

```diff
@@ -1,22 +1,56 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// A <see cref="PayloadConvention"/> that uses <see cref="SystemTextJsonPayloadSerializer"/>.
+    /// An implementation of the <see cref="PayloadConvention"/> class for System.Text.Json".
     /// </summary>
+    /// <remarks>
+    /// This class makes use of the <see cref="JsonSerializer"/> serializer and the <see cref="Encoding.UTF8"/> encoder.
+    /// </remarks>
```

Comment:
Imports all the relevant functionality from the SystemTextJsonPayloadSerializer and Utf8PayloadEncoder types.